misc/cartesianToPolar.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+#include <math.h>
+
+const double pi = 3.141592653589793238462643383279502884;
+
+/**
+give as arguments to the executable two x and y coordinates
+outputs a polar coordinate
+**/
+int main() {
+    double x, y;
+    double r, theta, thetaFinal;
+    scanf("%lf %lf", &x, &y);
+    r = hypot(x, y);
```

Comment:
This code, from line# 15 to line 42 - 90% of the code is useless. All the if-else cases are simply void. 
only line# 14 and 43 are needed.